src/en/reference-hook-tools.md

```diff
@@ -527,19 +527,24 @@ the current workload status. Without arguments, it just prints the workload
 status value e.g. 'maintenance'. With `--include-data` specified, it prints
 YAML which contains the status value plus any data associated with the status.
 
-python:  
+python:
 ```python
 from charmhelpers.core.hookenv import status_get
 
 charm_status = status_get()
 ```
-bash:  
+bash:
 ```bash
 status-get
 
 status-get --include-data
 ```
 
+Use `--service` to get the overall status for the service to which the unit
```

Comment:
I am not. Both --service and --application are in the help text for status-set, but only --application is in the help text for status-get.

Hmm.